### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by aath0** [aath0/MinoritiesInResearch#1]
# Roadmap

**The goal of this project is to evaluate how members of underrepresented minorities perceive research that directly concerns them.**

In this project, we plan to collectively design a questionnaire in order to quantify to which extend research that concerns members of geographical, cultural, gender, sexual or other minority groups view research done on them as ethical, useful and respectful.

Here is a (very preliminary) list of milestones for this project!

### Milestone: Gather a list of (broad) topics & design key-questions
> Clean up goals: which aspects of research do we want to test? What kind of research are we interested in: any kind of research? Only biological / psychological / social / * ?

* [# 3](https://github.com/aath0/MinoritiesInResearch/issues/3) Design some key directions that we want to address
* [# 4](https://github.com/aath0/MinoritiesInResearch/issues/4) Write up questions to target all these directions

### Milestone: Compile all questions together and proof-read them
> Before sending out the survey we got to make sure that all of our questions together make sense!

* Decide where to host the questionnaire + replies (ensure anonymity)
* Beta-test the questionnaire: Are questions clear and unambiguous?
* Is language inclusive?
* How long does it take to complete all questions?

### Milestone: Beta-test the questionnaire
> We will need specific feedback on the assembled survey to know whether the targets where the survey is aimed to feel that all important things are included and that nothing is missing?

* [# 10](https://github.com/aath0/MinoritiesInResearch/issues/10) Find beta testers who will provide critical feedback on the questionnaire

### Milestone: Submit an application to an ethical review board 
> We have to act ethically and get permission to run the survey
* Decide which review board to target
* Draft a cover letter and fill out application 

### Milestone: Send out the questionnaire to a broad audience
* Target people through social media / mailing lists / research groups
* Analyse responses and gather results!

**Comment by gedankenstuecke** [comment]
One thing that I find missing so far is a step that invites specific feedback on the beta-survey to know whether URM feel that all important things are included/nothing is missing?

**Comment by aath0** [comment]
Good point :smile: I have now added an additional step with beta testing!

**Comment by gedankenstuecke** [comment]
Looks good to me now!
